<?xml version="1.0" encoding="UTF-8"?>

<launch>

  <!-- These three nodes should be in GUI EMG Reset function. Do not start these nodes manually  -->

  <arg name="port" default="/dev/ttyACM0" doc="Port used by the Myo dongle. Check it with 'ls /dev/' and look for ttyACM0 for instance. You need to provide the full path." />
  <node pkg="ros_myo" name="myo_raw" type="myo-rawNode.py" args="$(arg port)" output="screen"/>
  <node pkg="arm_motion_controller_py3" name="world_to_myo_tf_publisher" type="world_to_myo.py"/>

  <!--<node pkg="arm_motion_controller_py3" name="emg_to_gripper" type="emg_to_gripper.py"/>  -->
  <node pkg="imu_human_pkg" name="emg_sum_node" type="emg_sum_node.py"/>  

</launch>
